src/containers/settings/EditServiceScreen.tsx

```diff
@@ -344,9 +344,17 @@ class EditServiceScreen extends Component<IProps> {
 
     if (proxy.isEnabled) {
       const serviceProxyConfig: IProxyConfig = service
-        ? stores.settings.proxy[service.id]
+        ? /*
+          TODO - [TS DEBT] find out why sometimes proxy[service.id] gives undefined
+          Note in proxy service id exist as key but value is undefined rather that proxy empty object
+
+          Temp fix - or-ed {} (to stores.settings.proxy[service.id] ) to avoid undefined proxy in settingStore throw error
+          */
+          stores.settings.proxy[service.id] || {}
         : {};
 
+      // @ts-ignore
+      console.log(stores.settings.proxy, service);
```

Comment:
Possibly worth removing this in order to avoid overloading the logs ;-)

Reply:
removed it and amended